packages/engine/src/scene/components/DirectionalLightComponent.ts

```diff
@@ -158,23 +159,93 @@ export const DirectionalLightComponent = defineComponent({
     useEffect(() => {
       if (!debugEnabled.value) return
 
-      const helper = new EditorDirectionalLightHelper(light.light.value)
-      helper.name = `directional-light-helper-${entity}`
-
-      const helperEntity = createEntity()
-      addObjectToGroup(helperEntity, helper)
-      setComponent(helperEntity, NameComponent, helper.name)
-      setComponent(helperEntity, EntityTreeComponent, { parentEntity: entity })
-      setVisibleComponent(helperEntity, true)
-
-      helper.layers.set(ObjectLayers.NodeHelper)
-
-      light.helperEntity.set(helperEntity)
+      if (!light.helper.value) {
+        const size = 1
+        const name = `directional-light-helper-${entity}`
+        const material = new LineBasicMaterial()
```

Comment:
let's put the materials and geometries in module scope as they're all going to be identitcal, and it will make the component a bit more readable with these verbose arrays